source/usage/other/craftscripts.rst

```diff
@@ -3,17 +3,26 @@ CraftScripts
 
 .. highlight:: javascript
 
-Scripts allow you to program small tasks without having to learn Java, figure out how to compile WorldEdit, or bother with reinventing the wheel. CraftScripts are written in JavaScript.
+CraftScripts allow you to write scripts with as little overhead as possible: no compiling WorldEdit, no setting up a plugins configuration file, etc. CraftScripts are written in JavaScript.
+
+The advantages of writing CraftScripts with WorldEdit are:
+
+* Hook right into WorldEdit's undo/redo system
+* Use WorldEdit's block place prioritization
+* Accept WorldEdit's powerful block type syntax (``//set sign[facing=north]``)
+* Get the region selected by the user
+
+.. note:: A basic understanding of both JavaScript and Java is recommended.
 
 Requirements
 ~~~~~~~~~~~~
 
-Before you start using CraftScripts, you'll have to install the `Rhino JavaScript engine <https://developer.mozilla.org/en-US/docs/Mozilla/Projects/Rhino>`_. A direct link to the download is `here <http://ftp.mozilla.org/pub/mozilla.org/js/rhino1_7R2.zip>`_. Open the zip file, and extract ``js.jar``. Note that if you download a newer version than the recommended one, the ``.jar`` file may have a different name. Move ``js.jar`` to the ``plugins/`` or ``plugins/WorldEdit`` folder (on Bukkit) or the ``mods`` folder (other platforms).
+Before you start using CraftScripts, you'll have to install the `Rhino JavaScript engine <https://developer.mozilla.org/en-US/docs/Mozilla/Projects/Rhino>`_. A direct link to the download is `here <http://ftp.mozilla.org/pub/mozilla.org/js/rhino1_7R2.zip>`_. Open the zip file, and extract ``js.jar``. Note that if you download a newer version than the recommended one, the ``.jar`` file may have a different name. Move ``js.jar`` to the ``plugins/`` or ``plugins/WorldEdit`` directory (on Bukkit) or the ``mods`` directory (other platforms).
```

Comment:
If you wouldn't mind updating the DL link to instead point to 1.7.12, that would be great.

As you can see at https://github.com/EngineHub/WorldEdit/blob/master/worldedit-core/build.gradle.kts#L26, that's the version that WorldEdit is currently bundling.

Reply:
Fixed in https://github.com/EngineHub/WorldEditDocs/pull/13/commits/f4440bf5baec9d16939e8b9c508a5fc9d841fc7b